alicloud/resource_alicloud_ga_endpoint_group.go

```diff
@@ -54,6 +54,12 @@ func resourceAliCloudGaEndpointGroup() *schema.Resource {
 				Computed:     true,
 				ValidateFunc: StringInSlice([]string{"HTTP", "HTTPS"}, false),
 			},
+			"endpoint_protocol_version": {
```

Comment:
=== RUN   TestAccAliCloudGaEndpointGroup_basic0
    provider_test.go:152: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudGaEndpointGroup_basic0 (857.52s)
PASS

=== RUN   TestAccAliCloudGaEndpointGroup_basic0_twin
    provider_test.go:152: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudGaEndpointGroup_basic0_twin (361.48s)
PASS

=== RUN   TestAccAliCloudGaEndpointGroup_basic1
    provider_test.go:152: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudGaEndpointGroup_basic1 (938.46s)
PASS

=== RUN   TestAccAliCloudGaEndpointGroup_basic1_twin
    provider_test.go:152: Skipping unsupported region eu-central-1. Supported regions: [cn-hangzhou]. Using cn-hangzhou as this test region
--- PASS: TestAccAliCloudGaEndpointGroup_basic1_twin (340.94s)
PASS